powershell-yaml.psm1

```diff
@@ -38,19 +38,43 @@ function Convert-ValueToProperType {
         if (!($Value -is [string])) {
             return $Value
         }
-        $culture = New-Object System.Globalization.CultureInfo("invariant")
```

Comment:
Makes sense. I was trying to be liberal in what it accepted, but if the spec isn't culturally aware, then that should be the behavior.